README.md

```diff
@@ -50,7 +55,47 @@ automatically [pip](https://pip.pypa.io/en/stable/) as well.
 a link to the swagger.json with the definition of the api.
 
 
-## CLI
+## Development
+
+### Test
+We are using Pytest](https://docs.pytest.org/en/latest/index.html) for tests. The tests are located in the package 
+`tests` and use the [conventions for python test discovery](https://docs.pytest.org/en/latest/goodpractices.html#test-discovery).
+
+To run the tests just execute:
+
+```
+python3 -m pytest -v
+```
```

Comment:
I am getting an error, any idea?
`$ python3 -m pytest -v
Traceback (most recent call last):
  File "/usr/local/Cellar/python/3.7.6_1/Frameworks/Python.framework/Versions/3.7/lib/python3.7/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/local/Cellar/python/3.7.6_1/Frameworks/Python.framework/Versions/3.7/lib/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/usr/local/lib/python3.7/site-packages/pytest.py", line 89, in <module>
    raise SystemExit(pytest.main())
  File "/usr/local/lib/python3.7/site-packages/_pytest/config/__init__.py", line 61, in main
    config = _prepareconfig(args, plugins)
  File "/usr/local/lib/python3.7/site-packages/_pytest/config/__init__.py", line 182, in _prepareconfig
    config = get_config()
  File "/usr/local/lib/python3.7/site-packages/_pytest/config/__init__.py", line 156, in get_config
    pluginmanager.import_plugin(spec)
  File "/usr/local/lib/python3.7/site-packages/_pytest/config/__init__.py", line 530, in import_plugin
    __import__(importspec)
  File "/usr/local/lib/python3.7/site-packages/_pytest/tmpdir.py", line 25, in <module>
    class TempPathFactory(object):
  File "/usr/local/lib/python3.7/site-packages/_pytest/tmpdir.py", line 35, in TempPathFactory
    lambda p: Path(os.path.abspath(six.text_type(p)))
TypeError: attrib() got an unexpected keyword argument 'convert'`

Reply:
Are you running it in the virtual environment?

Reply:
yes,

`
(.venv) MacBook-Pro-de-Rene:time-tracker-api moe$ python3 -m pytest -v
Traceback (most recent call last):
  File "/usr/local/Cellar/python/3.7.6_1/Frameworks/Python.framework/Versions/3.7/lib/python3.7/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/local/Cellar/python/3.7.6_1/Frameworks/Python.framework/Versions/3.7/lib/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/Users/moe/Documents/IOET/ernst-young/time-tracker-api/.venv/lib/python3.7/site-packages/pytest.py", line 89, in <module>
    raise SystemExit(pytest.main())
  File "/Users/moe/Documents/IOET/ernst-young/time-tracker-api/.venv/lib/python3.7/site-packages/_pytest/config/__init__.py", line 61, in main
    config = _prepareconfig(args, plugins)
  File "/Users/moe/Documents/IOET/ernst-young/time-tracker-api/.venv/lib/python3.7/site-packages/_pytest/config/__init__.py", line 182, in _prepareconfig
    config = get_config()
  File "/Users/moe/Documents/IOET/ernst-young/time-tracker-api/.venv/lib/python3.7/site-packages/_pytest/config/__init__.py", line 156, in get_config
    pluginmanager.import_plugin(spec)
  File "/Users/moe/Documents/IOET/ernst-young/time-tracker-api/.venv/lib/python3.7/site-packages/_pytest/config/__init__.py", line 530, in import_plugin
    __import__(importspec)
  File "/Users/moe/Documents/IOET/ernst-young/time-tracker-api/.venv/lib/python3.7/site-packages/_pytest/tmpdir.py", line 25, in <module>
    class TempPathFactory(object):
  File "/Users/moe/Documents/IOET/ernst-young/time-tracker-api/.venv/lib/python3.7/site-packages/_pytest/tmpdir.py", line 35, in TempPathFactory
    lambda p: Path(os.path.abspath(six.text_type(p)))
TypeError: attrib() got an unexpected keyword argument 'convert'
(.venv) MacBook-Pro-de-Rene:time-tracker-api moe$
`

Reply:
the you have a problem with your **python3** or **pytest** installation.
![image](https://user-images.githubusercontent.com/6514740/76634376-2b64d080-6514-11ea-85fe-3d9010be3a5b.png)

Reply:
Do you have env vars that need to be configured?

Reply:
Just in case. I solved this issue following these instructions:
https://stackoverflow.com/a/58189684/2697734

Reply:
I have `attrs==18.2.0` among my dependencies, but this is included probably by `pytest`.